utils/rate_limiter_test.go

```diff
@@ -77,9 +76,8 @@ func TestRateLimiterMaxBalance(t *testing.T) {
 func TestRateLimiterReconfigure(t *testing.T) {
 	rl := NewRateLimiter(1, 1.0)
 	assertBalance := func(expected float64) {
-		const precision = 0.0000001 // just some precision for comparing floats
-		delta := math.Abs(rl.balance - expected)
-		assert.Greater(t, precision, delta, "expect balance %v, was %v", expected, rl.balance)
+		const delta = 0.0000001 // just some precision for comparing floats
+		assert.InDelta(t, expected, rl.balance, delta)
```

Comment:
nit: can't we inline this assert after every test instead of having this be a fn?